src/Utils/TimingUtils/useCountdown.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import moment from 'moment';
+import 'moment-duration-format';
+
+function useCountdown(startTime, endTime) {
+  const [duration, setDuration] = useState(
+    moment.duration(endTime - startTime)
+  );
```

Comment:
from prev. pr:
![image](https://user-images.githubusercontent.com/67635802/109410667-168f8000-79a5-11eb-94cc-3aa33c393baa.png)

I believe you didn't understand me here,
talk to me about it.